app/socket-apis/__tests__/complete-round.js

```diff
@@ -0,0 +1,81 @@
+// https://github.com/EnCiv/civil-pursuit/issues/212
+const dturn = require('../../dturn/dturn')
+const completeRound = require('../complete-round')
+const { Discussions, insertStatementId, getStatementIds } = dturn
+
+const userId = '12345678abcdefgh'
+const synuser = { synuser: { id: userId } }
+
+beforeEach(async () => {
+  jest.spyOn(console, 'error').mockImplementation(() => {})
+
+  Discussions['discussion1'] = {
```

Comment:
please call dturn.initDiscussion and do what we can to use/test dtrun as much as possible. dturn has it's own readme if that helps: https://github.com/EnCiv/civil-pursuit/tree/master/app/dturn